package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@digicatapult/veritable-cloudagent",
-  "version": "0.8.8",
+  "version": "1.0.0",
```

Comment:
This should be 0.9.0 as its a pre v1 change.

Reply:
@mattdean-digicatapult Thoughts?

Reply:
my ticket says bump major version ...

Reply:
In Semver, prior to v1.0.0 a minor bump is considered a major version bump due to the rapidly changing APIs and structure.

https://semver.org/#doesnt-this-discourage-rapid-development-and-fast-iteration